homeassistant/components/apiai.py

```diff
@@ -0,0 +1,172 @@
+"""
+Support for API.AI webhook.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/apiai/
+"""
+import asyncio
+import copy
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import PROJECT_NAME, HTTP_BAD_REQUEST
+from homeassistant.helpers import template, script, config_validation as cv
+from homeassistant.components.http import HomeAssistantView
+
+_LOGGER = logging.getLogger(__name__)
+
+INTENTS_API_ENDPOINT = '/api/apiai'
+
+CONF_INTENTS = 'intents'
+CONF_SPEECH = 'speech'
+CONF_ACTION = 'action'
+CONF_ASYNC_ACTION = 'async_action'
+
+DEFAULT_CONF_ASYNC_ACTION = False
+
+DOMAIN = 'apiai'
+DEPENDENCIES = ['http']
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: {
+        CONF_INTENTS: {
+            cv.string: {
+                vol.Optional(CONF_SPEECH): cv.template,
+                vol.Optional(CONF_ACTION): cv.SCRIPT_SCHEMA,
+                vol.Optional(CONF_ASYNC_ACTION,
+                             default=DEFAULT_CONF_ASYNC_ACTION): cv.boolean
+            }
+        }
+    }
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
```

Comment:
I have just copied and modified ``alexa.py``. Maybe ``alexa.py`` is and old implementation?